src/main/java/core/basesyntax/GetArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface GetArea {
```

Comment:
please don't use verbs in class/interface names
```suggestion
public interface AreaCalculator {
```